backend/event/mutations/CreateEpisodeMutation.py

```diff
@@ -0,0 +1,45 @@
+from graphene import (
+    ID,
+    Field,
+    InputObjectType,
+    List,
+    NonNull,
+    ResolveInfo,
+    String,
+)
+
+from event.models import Episode
+from event.types.EpisodeType import EpisodeType
+from graphql_app.LettercraftMutation import LettercraftMutation
+from graphql_app.types.LettercraftErrorType import LettercraftErrorType
+from source.models import Source
+
+
+class CreateEpisodeMutationInput(InputObjectType):
+    name = String(required=True)
+    source = ID(required=True)
+
+
+class CreateEpisodeMutation(LettercraftMutation):
+    episode = Field(EpisodeType)
```

Comment:
I return the `episode` rather than `ok` here, because I need the ID to navigate the user to the new form page.